curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-c.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 64a5551d02815a7d323aab8a
+title: The Box Model Question C
+challengeType: 15
+dashedName: the-box-model-question-c
+--- 
+# --description--
+
+In the Elements pane, you can see the entire HTML structure of your page. You can click on any of the elements in this pane to select that specific element. Alternatively, you can click the blue-highlighted icon shown below on the left, and hover over any element on the page.
+
+![Inspector Icon](https://cdn.statically.io/gh/TheOdinProject/curriculum/594984d7c9f9e744577f19ea475b3864e8cc7c91/html_css/v2/foundations/inspecting-html-and-css/imgs/01.png)
+
+When an element is selected, the Styles tab will show all the currently applied styles, as well as any styles that are being overwritten (indicated by a strikethrough of the text). For example, if you use the inspector to click on the “Your Career in Web Development Starts Here” header on the TOP homepage, on the right-hand side you’ll see all the styles that are currently affecting the element, as seen below:
+
+![Overwritten style](https://cdn.statically.io/gh/TheOdinProject/curriculum/f8fd38fc62578d8e8368f5303126215a492847f0/foundations/html_css/inspecting-html-and-css/imgs/03.png)
+
+# --question--
+
+## --assignment--
+
+Play around with Chrome Dev Tools and see if you can answer the following question.
+
+## --text--
+
+Which feature in the Elements pane allows you to select any element on a webpage by hovering over it?
+
+## --answers--
+
+Blue-highlighted icon
+
+---
+
+Styles tab
+
+---
+
+Inspector tab
+
+---
+
+HTML structure view 
+
+
+## --video-solution--
+
+2
```

Comment:
```suggestion
1
```

<img width="396" alt="image" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/63889819/8b5e3a69-ed37-483d-9f71-50bcf3cd06b7">

Is this accurate? I can access the inspector tool from any tab, which is referred to as the "blue-highlighted" icon in the challenge text.